### DISCUSSION TITLE
1.19.2 dependency

### DISCUSSION THREAD

**Discussion by Povstalec** [cc-tweaked/CC-Tweaked#1422] [Q&A] [answered]
So, I already use a 1.19.3 version of CC:Tweaked as a dependency and I wanted to backport my mod to 1.19.2. However, when I switched my cctweaked dependency to a 1.19.2 version, I started getting the following error:

**Could not find org.squiddev:cc-tweaked-1.19.2:1.101.1_mapped_official_1.19.2.**

From testing I know that compiling a 1.19.3 cctweaked version on my 1.19.2 mod doesn't throw any gradlew errors, so I checked maven and could only find 1.19.3 and 1.19.4 versions listed. Am I just misunderstanding this, or is there no way to use versions below 1.19.2 as dependencies?

**Comment by SquidDev** [comment, marked as answer]
1.19.2 and before used a rather different Maven layout, so the 1.19.3+ instructions are not accurate there. The version specific branches [should have the correct instructions](https://github.com/cc-tweaked/CC-Tweaked/tree/mc-1.19.2#using).

**Reply by Povstalec** [reply]
Ah, I see. Thank you.
